### DISCUSSION TITLE
如何在VSCode中配置自己微调后的模型用于辅助编程

### DISCUSSION THREAD

**Discussion by SongHY-13** [unit-mesh/auto-dev-vscode#48] [Q&A] [answered]
在网上查找AI编程助手被介绍Autodev的文章吸引，因为可以使用开源的LLM，所以想要配置自己微调后的LLM，但是在Vscode下载后配置环节似乎没找到相关方式，开源LLM是指Tongyi?

**Comment by phodal** [comment, marked as answer]
只要用兼容  OpenAI  API  的模型就可以了

**Reply by SongHY-13** [reply]
感谢作者解答，我微调后的模型是兼容的，不过不清楚如何在开发环境中配置，请问在IDE中是选择engine切换，VScode中是在通过Override Base URL切换到自己的模型吗
![Autodev-vsc](https://github.com/unit-mesh/auto-dev-vscode/assets/62045743/c8871919-b3f7-4774-b2f5-061d1079543e)

**Reply by phodal** [reply]
对，更详细可以参考：https://vscode.unitmesh.cc/configuration.html
